autoload/ale/lsp/window.vim

```diff
@@ -0,0 +1,68 @@
+" Author: suoto <andre820@gmail.com>
```

Comment:
Move this code to a file like `autoload/ale/lsp_window.vim`. Everything in `ale/lsp/` and `ale/lsp.vim` should be only for implementing communication with language servers, and all of the code for how ALE handles the messages should be outside of that.

Reply:
Done!